### ISSUE TITLE
Preferences not supported

### ISSUE THREAD

**Issue by tomate-salat** [leaningtech/cheerpj-meta#76]
I tried the following code in your demo:

```java
import java.util.prefs.Preferences;

public class JavaFiddle
{
    public static void main(String[] args)
    {
        System.out.println("Compiling was ok");
        String value = Preferences.systemRoot().node("cheep").get("test", "nix");
        System.out.println("Value: " + value);
        System.out.println("I actually never make it to here :-(");
    }
}
```

I can see "Compiling was ok" but then it stops.

**Comment by alexp-sssup** [comment]
Confirming that we can reproduce the bug, please note that preferences would anyway not access the ones from the host system for obvious security reasons, but CheerpJ should virtualize the Preferences like the rest of filesystem accesses.

**Comment by tomate-salat** [comment]
> but CheerpJ should virtualize the Preferences like the rest of filesystem accesses.

That was actually the behavior I'd expected when I tested it.

**Comment by alexp-sssup** [comment]
This is fixed as part of the upcoming CheerpJ 3.0 release
